CHANGES

```diff
@@ -42,6 +42,12 @@ Vernacular Commands
 - Nested proofs may be enabled through the option `Nested Proofs Allowed`.
   By default, they are disabled and produce an error. The deprecation
   warning which used to occur when using nested proofs has been removed.
+- Added option Uniform Inductive Parameters which abstracts over parameters
+  before typechecking constructors, allowing to write for example
+  `Inductive list (A : Type) := nil : list | cons : A -> list -> list.`
+- Strengthened inference of conclusions of inductive constructos.
+  Inference of arguments no longer assumes that implicit parameters
+  are uniform.
```

Comment:
Maybe you can make it even clearer that this is a source of incompatibility and people are encouraged to use the above option to recover the previous behavior... (Is this a backward compatible way of patching failing code?)